index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+      integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Portfolio Setup and Mobile first</title>
+  </head>
+  <body>
+    <header class="logo-menu">
+      <div class="logo" onclick="window.location.href='index.html'">
+        <a href="index.html">My Logo</a>
+      </div>
+      <nav class="fa fa-bars bars"></nav>
+    </header>
+    <main>
+      <div class="main-container">
+        <section class="titleText">
+          <div class="title">
+            <h1>I’m Mariane Glad to see you!</h1>
+          </div>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+        </section>
+        <section>
+          <div class="connect">
+            <h2>LET'S CONNECT</h2>
+          </div>
+          <div id="footer-icon" class="footer-icons">
+            <li>
+              <a href="#footer-icon">
+                <img src="./images/twitter (2).svg" alt="twitter logo" />
+              </a>
+            </li>
+            <li>
+              <a href="#footer-icon">
+                <img src="./images/Linkedin icon.svg" alt="linkedin" />
+              </a>
+            </li>
+            <li>
+              <a href="#footer-icon"
+                ><img src="./images/m-icon.svg" alt="m logo"
```

Comment:
- [ ] As suggested by the previewer you should use an unordered list here (ul, li), Good job trying :+1: . here is the structure you should use to make it semantically correct:

```
<ul>
<li><a src=''><img></a></li>
<li><a src=''><img></a></li>
...............
</ul>
```

Reply:
implemented